backend/dal/Migrations/20210208235135_v01.07.03.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Something fishy is going on here since this is a v01.07.03 migration but all of your scripts are in v01.07.02 folders.

Reply:
just fixed the folder issue, I think my script was in 2 migration folders and I forgot to delete those scripts from the old folder